src/core/Akka.Tests/Pattern/BackoffSupervisorSpec.cs

```diff
@@ -172,7 +172,7 @@ public void BackoffSupervisor_must_support_default_stopping_strategy_when_using_
             // TODO: use FilterException
             EventFilter.Exception<TestException>().Expect(1, () =>
             {
-                var supervisor = Create(OnStopOptions().WithDefaultStoppingStrategy());
+                var supervisor = Create(OnStopOptions().WithDefaultStoppingStrategy().WithManualReset());
```

Comment:
If a GC happened, supervisor might be delayed and auto reset might kick in, resetting the restart counter to 0, failing the test